.github/workflows/build.yaml

```diff
@@ -82,3 +93,41 @@ jobs:
       with:
         name: Widelands ${{ matrix.config }} ${{ matrix.arch }} Installer
         path: ${{ github.workspace }}\Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.arch }}.exe
+
+  macos:
+    needs: [clang_tidy]
+    strategy:
+      matrix:
+        config:
+          - release
+          - debug
+        compiler:
+          - clang
```

Comment:
Are you planning to add more compilers sometime?

Reply:
Yes. Simply adding "gcc" to the matrix isn't sufficient though, so I skipped it in this branch.